cargo-near/src/build_extended/build_script.rs

```diff
@@ -100,6 +101,13 @@ impl<'a> BuildScriptOpts<'a> {
         } else {
             ":"
         };
+        if let Some(ref version_mismatch) = artifact.cargo_near_version_mismatch {
+            print_warn!(
+                version,
+                "WARNING: `cargo-near` version was coerced during build: {}",
```

Comment:
Please, make the warning more actionable, e.g.:

"WARNING: `cargo-near` crate version (potentially used in build.rs) did not match `cargo-near` build environment. Use the matching `cargo-near` CLI version or update `cargo-near` in Cargo.toml"